pkg/gds/emails/client.go

```diff
@@ -325,3 +328,160 @@ func (m *EmailManager) SendDeliverCertificates(vasp *pb.VASP, path string) (sent
 
 	return sent, errs.ErrorOrNil()
 }
+
+// SendExpiresAdminNotification sends the admins a notice that an identity certificate
+// will be expiring soon. This allows the admins to determine if a new review of the
+// TRISA member is necessary before the reissuance process begins.
+func (m *EmailManager) SendExpiresAdminNotification(vasp *pb.VASP, reissueDate time.Time) (sent int, err error) {
+	// Create the template context
+	ctx := ExpiresAdminNotificationData{
+		VID:                 vasp.Id,
+		CommonName:          vasp.CommonName,
+		Endpoint:            vasp.TrisaEndpoint,
+		RegisteredDirectory: vasp.RegisteredDirectory,
+		Reissuance:          reissueDate,
+		BaseURL:             m.conf.AdminReviewBaseURL,
+	}
+
+	if vasp.IdentityCertificate != nil {
+		// TODO: ensure the timestamp format is correct
+		ctx.SerialNumber = strings.ToUpper(hex.EncodeToString(vasp.IdentityCertificate.SerialNumber))
+		ctx.Expiration, _ = time.Parse(time.RFC3339, vasp.IdentityCertificate.NotAfter)
+	}
+
+	msg, err := ExpiresAdminNotificationEmail(
+		m.serviceEmail.Name, m.serviceEmail.Address,
+		m.adminsEmail.Name, m.adminsEmail.Address,
+		ctx,
+	)
+	if err != nil {
+		return 0, err
+	}
+
+	if err = m.Send(msg); err != nil {
+		return 0, err
+	}
+
+	return 1, nil
+}
+
+// SendReissuanceReminder sends a reminder to all verified contacts that their identity
+// certificates will be expiring soon and that the system will automatically reissue the
+// certs on a particular date.
+func (m *EmailManager) SendReissuanceReminder(vasp *pb.VASP, reissueDate time.Time) (sent int, err error) {
```

Comment:
If we can get this merged in by Wednesday, sending the reminders may be as simple as pressing a button :-)